document/core/exec/numerics.rst

```diff
@@ -180,13 +180,13 @@ When a number is stored into :ref:`memory <syntax-mem>`, it is converted into a
 Again these functions are invertable bijections.
 
 
-.. index:: SIMD, shape
+.. index:: vector numbers, shape
 .. _aux-lanes:
 
-SIMD
-....
+Vector numbers
+..............
 
-SIMD values have the same underlying representation as an |i128|. They can also be interpreted as a sequence numeric values packed into a |V128| with a particular |shape|.
+Vector numbers have the same underlying representation as an |i128|. They can also be interpreted as a sequence numeric values packed into a |V128| with a particular |shape|.
```

Comment:
Not part of your change but I noticed this typo: "sequence numeric values" => "sequence of numeric values"

Reply:
Fixed.